project/BuildDependencies/scripts/0_package.list

```diff
@@ -40,12 +38,14 @@ libxml2-2.7.8_1-win32.7z
 libxslt-1.1.26_1-win32.7z
 libyajl-2.0.1-win32.7z
 libzlib-vc100-1.2.5-win32.7z
-mysqlclient-6.1.3-win32-vc120.7z
-pcre-8.34-win32-vc120.7z
-PIL-1.1.7p-win32.7z
-python-2.7.10-win32.7z
+mysql-connector-c-6.1.6-win32-vc140.7z
+openssl-1.0.2e-win32-vc140.7z
+pcre-8.37-win32-vc140.7z
+pillow-3.1.0-win32-vc140.7z
+python-2.7.11-win32-vc140.7z
```

Comment:
This reminds me of  a similar crash I had months ago. The problem back then was that a python dll from the PATH was loaded. I don't think that this is the case here, but who knows.